src/formatters/SelectCellFormatter.tsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
+import classNames from 'classnames';
 
 export interface SelectCellFormatterProps {
   value: boolean;
+  disabled?: boolean;
   onChange(value: boolean, isShiftClick: boolean): void;
 }
 
-export function SelectCellFormatter({ value, onChange }: SelectCellFormatterProps) {
+export function SelectCellFormatter({ value, disabled = false, onChange }: SelectCellFormatterProps) {
   function handleChange(e: React.ChangeEvent<HTMLInputElement>) {
     onChange(e.target.checked, (e.nativeEvent as MouseEvent).shiftKey);
   }
 
   return (
-    <label className="rdg-checkbox-label">
+    <label className={classNames('rdg-checkbox-label', { disabled })}>
```

Comment:
`disabled` is too much of a generic classname, and it might create conflicts.
```suggestion
    <label className={classNames('rdg-checkbox-label', { 'rdg-checkbox-label-disabled': disabled })}>
```